modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -52,6 +52,25 @@ Even if your projects have not been identified as requiring action, you can proa
 [fix-gov-issues]
 == Fix Governance Conformance Issues
 
+When your API fails conformance to rulesets your organization validates against, you get an Email notification:
+
+. Click *Fix me* on your email to be redirected to the Exchange API asset Conformance Status page.
+.. If the latest version of your API has an editable branch, a pop-up appears where you can see the missing rulesets. Click *Add all rulesets*. +
```

Comment:
Before this we need to explain that from the Email, the user has also the possibility to go directly to API Designer using the second link with text _**Code in Design Center**_

Reply:
We should also add that you can click on `Edit criteria` and modify the API metadata to search which rulesets the project should conform to based on the specified fields

Reply:
Also specify that in the search, the retrieved rulesets will be based on all the conditions specified. Available filters are: Tags, Categories and Lifecycle State.